src/components/queryExpressionEditor/QueryExpressionEditor.scss

```diff
@@ -0,0 +1,24 @@
+.container {
+    overflow: auto;
+}
+
+.leftContainer {
+    float: left;
```

Comment:
Not sure about the use of float here, we tend to prefer the use of flex-boxes

Reply:
`.left` class in vapor can be used if you really need to float